### DISCUSSION TITLE
How do I create a layout for "root" routes?

### DISCUSSION THREAD

**Discussion by tscpp** [solidjs/solid-start#846] [Q&A] [answered]
I want to create a layout for the root routes (`/`, `/foo`, `/bar`). How can I achieve this using the solid router? Is it possible to mix `FileRoutes` with normal `Route`s?

**Comment by boehs** [comment, marked as answer]
You can do the normal outlet syntax, but wrap the folder in ()

ie: `routes/(layout)/foo.tsx`

And the file in routes would be named `(layout).tsx`
